draft-ietf-ppm-dap.md

```diff
@@ -1433,20 +1436,25 @@ never be the case that some reports are completed and others are not.)
 
 The Helper begins an aggregation job when it receives an `AggregationJobInitReq`
 message from the Leader. For each `PrepareInit` conveyed by this message, the
-Helper attempts to initialize VDAF preparation (see {{Section 5.1 of !VDAF}})
-just as the Leader does. If successful, it includes the result in its response
-that the Leader will use to continue preparing the report.
+Helper attempts to asynchronously initialize VDAF preparation (see
+{{Section 5.1 of !VDAF}}) just as the Leader does. If successful, it includes
+the result in its response that the Leader will use to continue preparing the
+report.
 
-To begin this process, the Helper checks if it recognizes the task ID. If not,
-then it MUST abort with error `unrecognizedTask`.
+Upon receipt of an `AggregationJobInitReq`, the Helper checks if it recognizes
+the task ID. If not, then it MUST abort with error `unrecognizedTask`.
 
 Next, the Helper checks that the report IDs in
 `AggregationJobInitReq.prepare_inits` are all distinct. If two preparation
 initialization messages have the same report ID, then the Helper MUST abort with
 error `invalidMessage`.
 
+If the Helper finds the `AggregationJobInitReq` to be valid, it immediately
+responds with HTTP status 201 Created.
+
 The Helper is now ready to process each report share into an outbound prepare
-step to return to the Leader. The responses will be structured as follows:
+step to return to the Leader. Asynchronously from any further requests from the
+Leader, the following structures are computed:
```

Comment:
This transition is a little awkward I think. I like the first paragraph ("it immediately responds with ..."). Perhaps tack on something like: "Later on, the Leader will make send an HTTP GET to <blah blah blah> to get the response. The Helper constructs the response as follows."

Reply:
I rewrote this to match what I did for continuation
```
If the Helper finds the `AggregationJobInitReq` to be valid, it immediately
responds with HTTP status 201 Created. The Helper is now ready to process each
report share into an outbound prepare step, asynchronously from any further
requests from the leader.

The following structures are computed:
```

LMK if that's also awkward, then I can fix both :D

Reply:
Much better!